Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contests-filters/ContestFilters.tsx

```diff
@@ -53,25 +54,27 @@ const ContestFilters = ({ onFilterClick }: IContestFiltersProps) => {
         [ possibleFilters, onFilterClick ],
     );
 
-    const renderFilterItem = useCallback(
-        ({ id, name }: IFilter) => {
+    const getRenderFilterItemFunc = useCallback(
+        (type: FilterType) => ({ id, name }: IFilter) => {
             // TODO: investigate why filters change ids
```

Comment:
log this and reference an issue maybe?

Reply:
The comment is wrong. It should be deleted.